### PR TITLE
fixed issue 34. nextAuction function should be Internal.

### PR DESCRIPTION
**Pull request by patidarmanoj10**
1) fixed issue 34. nextAuction function should be internal. 
2) Added some extra test cases in place of nextAuction test cases as its no more public
3) Moved internal function (i.e nextAuction) below public.

### CODE REVIEW

**Comment by patidarmanoj10** [comment]
@rokso  please review once

File: contracts/monolithic.sol [hunk 1]
```diff
@@ -1331,10 +1356,9 @@ contract Auctions is Pricer, Owned {
         uint recentAuction = whichAuction(lastPurchaseTick);
         uint currAuc = currentAuction();
         uint totalAuctions = currAuc - recentAuction;
-        if (currAuc == 0) {
-            _startTime = dailyAuctionStartTime;
-        } else {
-            _startTime = (currAuc * DAY_IN_SECONDS / timeScale) + dailyAuctionStartTime;
+        _startTime = dailyAuctionStartTime;
+        if (currAuc > 1) {
```
Comment (filcap) [inline contracts/monolithic.sol]: Lets add single line comment for the if condition.  Also consider using `1 days` time units.  Consider removing that named constant.

File: test/auctions.js [hunk 2]
```diff
@@ -460,8 +400,9 @@ contract('Auctions', accounts => {
       assert.equal(heartbeat[6].valueOf(), web3.eth.getBalance(proceeds.address), 'Proceed balance is not correct')
       assert.equal(heartbeat[7].valueOf(), await auctions.currentTick(), 'Current tick is not correct')
       assert.equal(heartbeat[8].valueOf(), await auctions.currentAuction(), 'Current auction is not correct')
-      const nextAuction = await auctions.nextAuction()
-      assert.equal(heartbeat[9].valueOf(), nextAuction[0], 'Next auction start time is not correct')
+      const dailyAuctionStartTime = (await auctions.dailyAuctionStartTime()).toNumber()
+      let nextAuctionStartTime = dailyAuctionStartTime + (heartbeat[8].toNumber() * SECS_IN_A_DAY)
+      assert.equal(heartbeat[9].valueOf(), nextAuctionStartTime, 'Next auction start time is not correct')
```
Comment (filcap) [inline test/auctions.js]: Confirm that all the natspec comments in heartbeat are still accurate for return values?

**Comment by rokso** [comment]
Looking good to me

**Pull request merged**